tests/moe_ep/test_moe_ep_sm107_block_scaled_mega_multirank.py

```
@@ -0,0 +1,294 @@
+    w13 = torch.randn(2, 2 * INTERMEDIATE, HIDDEN, device="cuda", dtype=torch.bfloat16)
+    w2 = torch.randn(2, HIDDEN, INTERMEDIATE, device="cuda", dtype=torch.bfloat16)
+    pack = MoEWeightPack(w13=w13, w2=w2)
+    if quant_kind == "nvfp4":
+        (fc1_w, fc1_sf), (fc2_w, fc2_sf) = preprocess_sm107_nvfp4_mega_weights(
+            pack, intermediate_size=INTERMEDIATE, hidden_size=HIDDEN
+        )
+        assert fc1_w.shape == (2, HIDDEN // 2, 2 * INTERMEDIATE)
+        assert fc2_w.shape == (2, INTERMEDIATE // 2, HIDDEN)
+        assert fc1_sf.dtype == torch.float8_e4m3fn
+    else:
+        (fc1_w, fc1_sf), (fc2_w, fc2_sf) = preprocess_sm107_mxfp8_mega_weights(
```

Ruff (0.16.1)

[warning] Unpacked variable `fc2_sf` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)